four_wheel_steering_controller/CMakeLists.txt

```diff
@@ -0,0 +1,65 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(four_wheel_steering_controller)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+
+#Add custom (non compiling) targets so launch scripts and python files show up in QT Creator's project view.
+file(GLOB_RECURSE EXTRA_FILES */*)
```

Comment:
Please remove these changes, we shouldn't include qtcreator hacks in the cmake of released packages.

Reply:
agreed